src/utils/csv.ts

```diff
@@ -13,7 +13,7 @@ export const accountsToCSV = (accounts: DWAccountDisplay[], asset: Asset) => {
     dpath: account.pathItem.path,
     asset:
       (account.balance
-        ? parseFloat(
+        ? bigify(
             fromTokenBase(new BN(account.balance.toString()), asset.decimal).toString()
```

Comment:
Can this be a BigNumber or hex-string rather than a decimal string? Can we use `bigify` here?